apps/jobs/migrations/0011_submission_challenge_name.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hey, @SinghSatbir I think you misunderstood the issue, We wanted to only `display` the challenge name in the submissions table in Django admin.
There is no need to add a field for it and migrations.
Please feel free to ask if you still have doubts.